_posts/2019-05-30-hobby-dev.md

```diff
@@ -0,0 +1,53 @@
+So once the guide season was over, I went and found a job working in a climbing gym.
+
+In the 6 years in that industry, I can count on one hand the number of climbing trips I took purely for fun. And I could
+probably say nearly the same thing for backpacking (I _might_ need the other hand for this one). These things that used
+to be hobbies ceased to have the same draw. I was spending my time on the clock doing those things, I didn't want to
+spend my time _off_ the clock doing them, too. So I didn't.
+
+Fast forward to now, working as a web developer. I started out programming for fun and convenience, little things that
+made my life easier, but mostly were just for fun. Now that I spend 6-8 hours a day doing this, I go home and do just
+about anything _but_ programming for fun. This hobby of mine is now mostly dead.
+
+
+## Mostly dead is still slightly alive
+I was always good at my other jobs, I even eventually rose to management, but you know which people were _best_ at it?
+The people who still chose to maintain its hobby status. The best backpacking guides were the ones who practically
+lived in the woods. The best climbers were the ones spending their weekends at a crag somewhere, learning new skills
+to translate into the gym. I don't know that I have been working in this field long enough to say with 100% conviction
+that this is also true here. But I believe there are 2 types of people in this world: those who can extrapolate from
+incomplete data.
+
+It may be that I'm wrong for 90% of people in this job, but I think for me, with my history, and my situation,
+maintaining hobby status is one of the most important things I can do for my career. Being self taught, there wasn't
+necessarily a logical progression in learning. I hopped from problem I was trying to solve to new problem in whatever
+order I felt like. This left lots of gaps in my knowledge. Lots of questions I didn't even know I needed to ask. And
+while work has succeeded in filling in most of these gaps, I'm limited in what I can learn by the problems I'm trying
+to solve at work. And because there is often a deadline, I don't get to explore _every_ possible solution to a problem.
+I pick the best one I know and implement it. Then move on to the next problem.
+
+Hobbies are different. There isn't really a deadline. There aren't the same restrictions placed on you.
+You can afford to do it wrong _many_ times.  And each "wrong" attempt is not a failure, simply a learning
+experience for how _not_ to solve the problem. (Insert one of the many variations on Thomas Edison creating a
+light bulb in 1,000 steps/finding 1,000 ways _not_ to make a light bulb, you get the idea.) So as someone who
+started in this field as a hobby, the way to continue to progress in this field, for me to become the best I can
+be at it, I need to revive my hobby. After all, it's only mostly dead. And mostly dead is still slightly alive.
```

Comment:
I would really love to hear some kind of plan for a couple hobby projects, or something? Loved the voice in this, really want to hear a resolution to the conflict! Maybe in a pt 2?